linux/keyman-config/debian/control

```diff
@@ -1,66 +1,92 @@
 Source: keyman-config
-Maintainer: Keyman team <support@keyman.com>
-Section: python
+Section: utils
 Priority: optional
-Build-Depends: dh-python, debhelper (>= 9), bash-completion,
- help2man, python3-all (>= 3.5), python3-setuptools,
- python3-pip, python3-magic, python3-lxml, python3-numpy,
- python3-pil, python3-requests, python3-requests-cache,
- python3-gi, ibus, gir1.2-webkit2-4.0,
- python3-qrcode <!pkg.keyman-config.xenial>,
+Maintainer: Debian Input Method Team <debian-input-method@lists.debian.org>
+Uploaders:
+ Keyman team <support@keyman.com>,
+Build-Depends:
+ bash-completion,
+ debhelper (>= 11),
+ dh-python,
+ gir1.2-webkit2-4.0,
+ help2man,
+ ibus,
  python-qrcode <pkg.keyman-config.xenial>,
```

Comment:
Since we're dropping Xenial, do some of these dependencies get removed?

Reply:
Done